packages/devextreme-scss/images/icons/material/today.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please, add <title>today</title> inside svg tag. It should be presented.
Example:
<img width="711" alt="image" src="https://github.com/user-attachments/assets/0532521e-ffc1-4dc4-b9fe-3cb3d6e3e9b9" />

Reply:
Let's do it for three svg icons